content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -157,6 +157,13 @@ Starting with v1.22 and later, when creating a cluster with kubeadm, if the user
 the `cgroupDriver` field under `KubeletConfiguration`, kubeadm defaults it to `systemd`.
 {{< /note >}}
 
+{{< note >}}
```

Comment:
I wouldn't use a note here. Use a plain paragraph.

Reply:
OK, made it a plain (non-note) paragraph, as suggested